src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -24,8 +25,23 @@ internal class ProcessMetrics
     internal static readonly AssemblyName AssemblyName = typeof(ProcessMetrics).Assembly.GetName();
     internal static readonly Meter MeterInstance = new(AssemblyName.Name, AssemblyName.Version.ToString());
 
+    private const string MetricPrefix = "process.runtime.dotnet.";
+
     static ProcessMetrics()
     {
+        // TODO: change to ObservableUpDownCounter
+        MeterInstance.CreateObservableGauge(
+            $"{MetricPrefix}memory.usage",
+            () => Diagnostics.Process.GetCurrentProcess().WorkingSet64,
```

Comment:
Maybe the `Diagnostics.Process.GetCurrentProcess()` part can be pre-calculated and reused?